app/views/accounts/show.html.erb

```diff
@@ -70,8 +70,7 @@
   </div>
 <% end %>
 
-<% if TwoFactorAuthentication::WebauthnPolicy.new(current_user).platform_configured? &&
-    IdentityConfig.store.platform_authentication_enabled %>
```

Comment:
We don't need that check. I removed it because I want it to be visible for existing users still who may want to look their webauthn stuff in their account page. If we didn't remove it, when we disable the flag for new users we would make it so it wouldn't show the existing webauthn platform page.

Reply:
ahhhh, thanks!

Reply:
could we rename (or add) the configuration key to clarify registration vs enabled overall?